### DISCUSSION TITLE
How to scroll through an folder containing pdf files?

### DISCUSSION THREAD

**Discussion by bekaa02** [sumatrapdfreader/sumatrapdf#2438] [Q&A]
I am a newbie in sumatra-PDFviewer. I want to scroll through a folder containing one hundred of small PDF files opened for reading. I want to do this in a manner similar to thumb through a real book, i.e., not only to scroll the directory of the folder alone.  How can I open the folder and the included PDF files so that I can scroll through the folder along all PDF file members? Up to now I succeeded only in opening one single PDF-file of the folder. As a result, the file counter has only the value 1/1 and, correspondingly,  i cannot scroll through the folder by using the left and right arrows.
Is there a way to manage this?

**Comment by GitHubRulesOK** [comment]
CTRL SHIFT right and left will attempt to open next file in current tab (including drag and drop a folder, which is usually best), however will stop if it hits an error file, so beware some zip types or other file it may attempt in a folder by scanning from the open file and stumble upon opening.
